.github/workflows/dependent.yml

```
@@ -0,0 +1,75 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: '/'
+    schedule:
+      interval: monthly
+    open-pull-requests-limit: 10
+    reviwers: 
+      - "divyarajmasani"
+
+  - package-ecosystem: npm
+    directory: '/'
+    schedule:
+      interval: monthly
+    open-pull-requests-limit: 10
+    versioning-strategy: increase
+    reviwers: 
+      - "divyarajmasani"
+    groups:
+      production-version:
+        dependency-type: "production"
+        applies-to: version-updates
+        update-types:
+          - "minor"
+          - "patch"
+      production-security:
+        dependency-type: "production"
+        applies-to: security-updates
```

Thank you for the PR,

On other thought creating these many groups sounds over optimising, especially for the packages we use.

Let's trim it down to just two groups, `production` and `development` for minor updates. And raise separate PR for  major versions as it may have breaking changes. 

```suggestion
      production:
        dependency-type: "production"
        update-types:
          - "minor"
          - "patch"
```

---

- [x] Reduced the groups